### ISSUE TITLE
Using Compactor for faster compression

### ISSUE THREAD

**Issue by YayJohn** [wabbajack-tools/wabbajack#1239]
compression can be faster

why make compression slower when it can be faster

use this https://github.com/Freaky/Compactor, it claims its faster than using the compact.exe file

faster compression times cuz why not

alternatives are compact.exe and compactgui

but i think compactgui isnt any faster than compact.exe

I'm a beginner programmer so I have no real idea if it is possible to integrate this program into wabba

**Comment by Unnoen** [comment]
Wabbajack uses the same method as those tools. 

There may be ways to improve detection of files that wouldn't benefit much or any from compression, but otherwise WJ is doing the same thing.

**Comment by Unnoen** [comment]
Actually, my mistake, looking into this more the methods are different, it may be worth looking into this more.

**Comment by halgari** [comment]
I looked into this same method when I implemented compaction in Wabbajack. Compact.exe calls these same methods, and if you use the API directly without a bunch of extra work, it will simply mark the file as "ready to be compacted" and then Windows uses all your cores and all your disk bandwidth to compress the files. So yes it's faster but it also renders your system unusable while you wait. 

So it was simpler and more stable to use the (slower) method of letting compact.exe compact at a slower rate.

TL;DR - they both use the same methods, the APIs are async by default, making them sync is tricky
